airbyte-ci/connectors/pipelines/pipelines/airbyte_ci/connectors/test/steps/common.py

```diff
@@ -594,6 +632,7 @@ async def _run(self, connector_under_test_container: Container) -> StepResult:
         try:
             self._validate_job_can_run()
         except AssertionError as exc:
+            self.logger.info(f"Could not run live tests for {self.context.connector.technical_name} due to validation error {str(exc)}.")
```

Comment:
Let me know if I'm mistaken:
* in this PR you're making live test run for `airbyte-ci connectors test` invocation.
* if `_validate_job_can_run` raises an assertion error we fail the step.

I think it means that:
- destinations will always fail CI as they can't run live tests
- any connector without a `testConnections` will fail 

Am I right? 
Shall we skip the step instead of failing it when validation fails?

Reply:
Yep that's right. Skipping instead makes sense to me.